Resources/Prototypes/DeltaV/Reagents/medicine.yml

```diff
@@ -0,0 +1,63 @@
+- type: reagent
+  id: Testosterone
+  name: reagent-name-testosterone
+  group: Medicine
+  desc: reagent-desc-testosterone
+  physicalDesc: reagent-physical-desc-hormonal
+  flavor: bitter
+  color: "#eb692d"
+  boilingPoint: 1321.0
+  meltingPoint: 424.1
+  metabolisms:
+    Medicine:
+      metabolismRate: 0.02
+      effects:
+      - !type:GenericStatusEffect
+        key: Masculinized
+        component: Masculinized
+        type: Add
+        time: 2
+        refresh: false
```

Comment:
make an effect to add HormoneComponent with a target sex
then just
```yml
- !type:HormoneEffect
  target: Male
```

Reply:
are you wanting me to implement the temporary status effect logic myself for these components? AFAICT the `GenericStatusEffect` component only takes a single key and not a full component description

Reply:
yes reimplement it

ideally generic status effect would use a ComponentRegistry but oh well